scripts/tests/versioner_hashes.yaml

```diff
@@ -189,9 +189,15 @@ d5ed287fadcea6167a6c9d75924f1c11007ab51d: #26383
   libmesh: bad5b94
   moose-dev: f8a2f21
   wasp: 91987a3
-e6ada94b4f7b638bab0da284abc02315fcf63f3e: #26462
+668ed3ad53d35c108044f8f742507e07a278d48f: #26462
   mpich: 702900f
   petsc: b32e40b
-  libmesh: 7cd53c3
-  moose-dev: 4b9bf44
+  libmesh: 376999c
+  moose-dev: 82b4336
+  wasp: 91987a3
```

Comment:
I don't follow... I am changing everything (mpich 4.0.2 -> 4.1.2), so everything will change (except wasp).

But I rebased against Next, and this is what I got. What am I supposed to rebase against? Do I have to cherry-pick around your merged branch or something?

Reply:
This is the entry from #26462, not this PR (which you added in Line 198 and after). Your PR shouldn't change this entry. Since my changes are now in next, you should remove commits [939a8be](https://github.com/idaholab/moose/pull/26566/commits/939a8beb511feb69b8196f87ebacbb1c1b561ddb) and [2e82820](https://github.com/idaholab/moose/pull/26566/commits/2e828204d1f82e1567aecb3de8f50f70983164b8), and then re-confirm that your build changes in [cfdb064](https://github.com/idaholab/moose/pull/26566/commits/cfdb064b98265784f7070352f9bc2d9a33fb7e11) are as you expect. The fact that the first two commits I cite above worked cleanly suggests that your rebase was on a slightly older next.

Reply:
This has been resolved through an offline conversation and interactive rebase shenanigans.